client/signup/config/flows-pure.js

```diff
@@ -13,28 +13,6 @@ const noop = () => {};
 const getUserSocialStepOrFallback = () =>
 	isEnabled( 'signup/social-first' ) ? 'user-social' : 'user';
 
-const getP2Flows = () => {
-	return isEnabled( 'p2-enabled' )
```

Comment:
This `p2-enabled` feature flag can actually be removed from a few more files as well.

Reply:
Cleaned a few more in [12326b6](https://github.com/Automattic/wp-calypso/pull/101712/commits/12326b640a7abd39c777f8dd7af5ae3c1a617e58).